mumbaihackathon_in/www/index.html

```diff
@@ -1,6 +1,6 @@
 {% extends "templates/hackathon.html" %}
 
-{% block title %}Mumbai Hackathon 2020 - An Open-source Hackathon{% endblock %}
+{% block title %}Mumbai Hackathon {{year}} - An Open-source Hackathon{% endblock %}
```

Comment:
In registration, the year is computed from the current year, but this comes from settings. Both should be consistent.